### ISSUE TITLE
NVDA crashes and stops responding in the Finereader source initialization failed error dialogue box

### ISSUE THREAD

**Issue by enessaribas** [nvaccess/nvda#14774]
### Steps to reproduce:
1. In Abby Finereader,  right arrow to scan, activate scan button, and right arrow to PDF.  Do this while there is no scanner connected.  For Finereader 13 or 14, additionally tab to the quality, and change the value. These steps trigger this error dialogue.
### Actual behavior:
As soon as focus moves tyo the dialogue box and NVDA reads it, NVDA stops responding. This is a total freeze and crash of NVDA.  NVDA will eventually restart itself. But restarting NVDA doesn't resolve the crash.  Moreover about a minute later, and multiple restarts of NVDA, NVDA causes Finereader to stop responding.

### Expected behavior:
  NVDA will read the dialogue normally and allow activation of the OK button without any crash.

### NVDA logs, crash dumps and other attachments:
attached NVDA log
[nvda.log](https://github.com/nvaccess/nvda/files/11130839/nvda.log)

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
alpha-27981,51aacac6 
#### Windows version:
Windows 11 22h2
#### Name and version of other software in use when reproducing the issue:
Abby Finereader version 16, release 3 update 1, but occurs at least as far back as Finereader 14.
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
this issue has existed for multiple years and is still present
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by XLTechie** [comment]
@Adriani90 wasn't an issue very similar to this just closed recently? I could be 
mistaken, but it was this same software.

**Comment by Adriani90** [comment]
I cannot reproduce this with ABBYY Finereader 15 or 16 when no scanner is connected. In fact, in that case I cannot do anything after activating "PDF" because a gialog pops up saying that no scanner is connected. So it is not possible to change the quality.
I will test with a scanner connected later on.

However, I strongly believe this is related to a synth driver, I think the IBM TTS driver. I also see a deprecated app module in your log.

Please try following:
1. Press windows+r
2. paste this command
"nvda -c %tmp%\nvda_finereaderTest"
3. Confirm the dialogs
4. Don't change any settings apart from the settings in the settings ring

Does this issue occur when you ust this test version of NVDA?

After testing, you can delete the nvda_finereaderTest folder in %temp%.

**Comment by jcsteh** [comment]
It looks like the application (or at least the window in question) actually stops responding. Thus, it's quite likely that #14759 will help with this. I'd suggest giving the build there a try. It might not necessarily allow you to access what you want if the app is indeed hanging, but it'll at least prevent NVDA from hanging with it.

**Comment by seanbudd** [comment]
There's a new alpha build which incorporates this fix: https://ci.appveyor.com/project/nvaccess/nvda/build/alpha-28022,5f340be5
do you mind testing with it?

**Comment by enessaribas** [comment]
This not only resolves the issue, it also stopped finereader from crashing.

**Comment by seanbudd** [comment]
Great, closing as fixed
